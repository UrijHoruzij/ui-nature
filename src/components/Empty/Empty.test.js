import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { Empty, Icon } from "../";
import ThemeProvider from "../../ThemeProvide";

describe("Empty component", () => {
  test("Render empty", () => {
    const component = (
      <ThemeProvider>
        <Empty image={<Icon name="close" />} description="Файл не найден" />;
      </ThemeProvider>
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
