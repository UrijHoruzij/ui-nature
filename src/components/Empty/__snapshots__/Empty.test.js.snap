// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty component Render empty 1`] = `
Array [
  .c3 {
  font-family: Montserrat,sans-serif;
  border: none;
  outline: none;
  font-weight: 500;
  color: #333333;
  text-transform: none;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c4 {
  font-size: 16px;
  line-height: 20px;
  font-weight: 400;
}

.c2 {
  stroke: #333333;
  fill: #333333;
  -webkit-transition: all .3s ease-in-out;
  transition: all .3s ease-in-out;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120px;
  max-height: 120px;
}

.c5 {
  margin: 0;
  margin-top: 8px;
}

<div
    className="c0"
  >
    <div
      className="c1"
    >
      <svg
        className="c2"
        fill="none"
        height="16"
        name="close"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M2 2L14 14"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M14 2L2 14"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </div>
    <p
      className="c3 c4 c5"
      color="text"
      type="text"
    >
      Файл не найден
    </p>
  </div>,
  ";",
]
`;
